managed/services/checks/funcs.go

```diff
@@ -130,7 +130,7 @@ func ipIsPrivate(args ...interface{}) (interface{}, error) {
 		_, net, err := net.ParseCIDR(ip)
 		if err != nil {
 			log.Errorf("invalid ip/network address: %q", ip)
-			return nil, nil
+			return nil, errors.Errorf("invalid ip/network address: %q", ip)
```

Comment:
@artemgavrilov It seems like it's legit to return an error in this one case. WDYT?

Reply:
This behavior was intentional as it's described if func description. I don't know why it was done like this, but this function was made for advisors (it used by advisors itself), so most likely we can't change it just like this. It may break existing advisors that rely on such output.

Reply:
Yes, it actually breaks the tests, so I reverted.